Kernel/Arch/x86/common/Processor.cpp

```diff
@@ -1614,7 +1642,9 @@ extern "C" void enter_thread_context(Thread* from_thread, Thread* to_thread)
     VERIFY(in_critical > 0);
     Processor::restore_in_critical(in_critical);
 
-    if (has_fxsr)
+    if (has_xsave_avx_support)
+            asm volatile("xrstor %0" ::"m"(to_thread->fpu_state()));
```

Comment:
This doesn't look like it's indented correctly :)

Reply:
Looks like `clang-format`'s done me dirty here!